### PR TITLE
Load imgur links directly, also put a rounded border on comment flair.

### PR DESCRIPTION
**Pull request by Protuhj**
I was tired of being taken to imgur's mobile site (loads like crap on my phone), so I made single-image links go directly to the image itself, rather than the mobile site.  

Also put a rounded-border on comment flair (in my opinion, it looks fine in light and dark themes).

### CODE REVIEW

File: res/drawable/rounded_corner.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <padding
+            android:bottom="2dp"
+            android:left="4dp"
+            android:right="4dp"
+            android:top="2dp"/>
+
+    <!-- Here is the corner radius -->
+    <corners android:radius="5dp"/>
+
+</shape>
```
Comment (zagaberoo) [inline res/drawable/rounded_corner.xml]: Missing newline at end of file

File: res/layout/comments_list_item.xml [hunk 2]
```diff
@@ -163,9 +163,8 @@
                 android:singleLine="true"
                 android:ellipsize="marquee"
                 android:textAppearance="?android:attr/textAppearanceSmall"
-                android:textColor="@color/gray_50"
-                android:visibility="gone"
-                />
+					  android:visibility="gone"
+					  android:background="@drawable/rounded_corner" android:textColor="@color/black"/>
```
Comment (zagaberoo) [inline res/layout/comments_list_item.xml]: This should probably be broken up into two lines.

**Comment by zagaberoo** [comment]
I'm a little wary of the imgur change, but I don't think it's unreasonable if you gate it on an off-by-default preference.

**Comment by zagaberoo** [comment]
Long time no see!  Be sure to let me know when the this PR is ready for me to take another look.

**Comment by Protuhj** [comment]
I've been updating the project for personal purposes, I probably should just open new pull requests.

**Comment by zagaberoo** [comment]
Ah that makes sense given that from github's perspective a branch is a pull request.

I'll close this one, feel free to open others.

Also, regarding my previous comment on the imgur integration, having it guarded by an on-by-default preference would be fine too as long as users can disable it.

**Pull request closed without merging**